.github/workflows/ci_julia_nightly.yml

```diff
@@ -0,0 +1,50 @@
+name: CI (Julia nightly)
```

Comment:
Why does this need to be a separate file?

Reply:
It's the only way to get a status badge that is green even if tests fail on Julia nightly.

If you don't mind a status badge that turns red if tests fail on Julia nightly but pass on all other Julia versions, then we can merge those two workflow files into a single workflow file.

Reply:
Alright, thanks. Might as well keep it like this then.